src/components/shared/ListItem.vue

```diff
@@ -93,6 +96,13 @@ export default {
       default: null
     }
   },
+  methods: {
+    divideIdValue (id) {
+      const arr = id.split('')
+      arr.splice(0, arr.length / 2)
```

Comment:
@vickywane Dividing the text length by two is not an optimal solution. The value can be much longer than it is now in the future (e.g. a 500 char public key ), but that should not break the page. 

The good news however is that due to your latest use vuetify layout markup, I think you can just get rid of this `divideIdValue` function and everything should still look fine.